pkgs/servers/sql/mariadb/default.nix

```diff
@@ -50,8 +50,8 @@ common = rec { # attributes common to both builds
   cmakeFlags = [
     "-DBUILD_CONFIG=mysql_release"
     "-DMANUFACTURER=NixOS.org"
-    "-DDEFAULT_CHARSET=utf8"
-    "-DDEFAULT_COLLATION=utf8_general_ci"
+    "-DDEFAULT_CHARSET=utf8mb4"
```

Comment:
For those who wonder what the difference is: https://stackoverflow.com/questions/30074492/what-is-the-difference-between-utf8mb4-and-utf8-charsets-in-mysql